include/SugarObjects/templates/person/Person.php

```diff
@@ -376,6 +376,6 @@ public function setLawfulBasis($basis, $source)
         $this->date_reviewed = $date_test;
         $this->save();
 
-        return 0;
+        return 1;
```

Comment:
A suggestion: returning the return value of $this->save() (one line above) is more informative:
for e.g:
```
$ret = $this->save();
return $ret;
```
or:
```
return (bool)$ret;
```